sklearn/tests/test_site_joblib.py

```diff
@@ -26,24 +16,4 @@ def test_old_pickle(tmpdir):
             b'\x0fU\nallow_mmapq\x10\x88ub\x01\x00\x00\x00\x00\x00\x00\x00.',
             mode='wb')
```

Comment:
Essentially this becomes a test of joblib. It's still useful, but somewhat non standard.